src/charts/CytoViz/CytoViz.js

```diff
@@ -128,7 +157,24 @@ export const CytoViz = (props) => {
       });
     }
   };
-
+  /**
+   * appends meta data needed for the visualisatation to the data field of the edges
+   * @param {*} elements in the Cityscape format
+   * @returns the elements object with fields: {_asInEdgeHighlighted: false, _asOutEdgeHighlight: false}
+   */
+  const appendMissingDataValues = (elements) => {
+    elements.forEach((element) => {
+      if (element.group === 'edges') {
+        if (!Object.prototype.hasOwnProperty.call(element.data, '_asInEdgeHighlighted')) {
```

Comment:
I think you can use more light syntax for this condition: `if (!('_asInEdgeHighlighted' in element.data))` :slightly_smiling_face: